apps/src/lab2/views/components/editor/CodeEditor.tsx

```diff
@@ -77,7 +83,7 @@ const CodeEditor: React.FunctionComponent<CodeEditorProps> = ({
     });
   };
   useEffect(() => {
-    if (editorRef.current === null || didInit) {
+    if (!fontSizeLoaded || editorRef.current === null || didInit) {
```

Comment:
This prevents a flicker caused by the setting of the `editorFontSizeKey` (initial value set in `lab2ViewRedux`).

Reply:
This could be a follow-up, but I wonder if it would make sense to move the fetching of relevant user preferences to the lab2 wrapper. We already [fetch and store permissions there](https://github.com/code-dot-org/code-dot-org/blob/c35de3e2ffd60254be6ed3b486f7b627560dfd17/apps/src/lab2/views/Lab2Wrapper.tsx#L50-L53). The thing to consider would be whether we only fetch user preferences per-lab or just fetch all the ones that lab2 could use on load.

Reply:
On interesting idea! I'll add this as a follow-up. Thanks!

Reply:
https://codedotorg.atlassian.net/browse/CT-1183

Reply:
Discussed over Slack...will incorporate this update in this PR because the fetching of user's preferred font size is causing a couple hiccups with `pythonlab` UI tests (need more time for CodeMirror to load since waiting for font size from backend).

Reply:
After looking into this a littel more deeply, I will hold off on making this change as I think it requires more careful thought/discussion. But about to push a small update to address the setting of the code editor loaded state AFTER the font size is fetched and updated.